### DISCUSSION TITLE
No event for click emitted in very simple example

### DISCUSSION THREAD

**Discussion by Septias** [emilk/egui#1559] [General]
Hi, somehow this does not fire a click-event. I want to have the whole screen clickable.

```rust
fn update(&mut self, ctx: &egui::Context, _frame: &mut eframe::Frame) {
    let resp = CentralPanel::default().show(&ctx, |ui| {
        ui.allocate_space(ui.available_size());
    });

    resp.response.interact(Sense::click());

    if resp.response.clicked() {
        info!("{:?}", ctx.pointer_latest_pos());
    }
}
```
what am I doing wrong?

**Comment by emilk** [comment]
`let response = resp.response.interact(Sense::click());`

https://docs.rs/egui/latest/egui/struct.Response.html#method.interact

**Reply by Mingun** [reply]
Probably you should mark this method with `#[must_use]`?

**Reply by Septias** [reply]
thank you that solves it :)
